src/hooks/useCioAutocomplete.ts

```diff
@@ -125,7 +126,7 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
     highlightedIndex,
     getInputProps,
     getItemProps: getItemPropsDownShift,
-  } = useDownShift({ setQuery, items, onSubmit, cioClient, previousQuery });
+  } = useDownShift({ setQuery, items, onSubmit, cioClient, previousQuery, onIsOpenChange });
```

Comment:
Could it be an issue that the `isOpen` value we're exposing _might be_ different from downshift's internal state? Since we return `isOpen: isOpen && items?.length > 0` there is a scenario where `onIsOpenChange` can trigger with `changes.isOpen` being true and our own `isOpen` as false.

Reply:
This is a great catch! Thank you for raising it

Reply:
`isOpen && items?.length > 0` was added to not show the dropdown if there are no items

I just got an ask from a customer to be able to show the dropdown even if there's no items.

I will create a separate story for that